common/internal_model/src/main/scala/uk/ac/wellcome/models/work/internal/Image.scala

```diff
@@ -0,0 +1,29 @@
+package uk.ac.wellcome.models.work.internal
+
+sealed trait BaseImage[+Id] extends HasIdState[Id] {
+  val id: Id
+  val location: DigitalLocation
+}
+
+sealed trait ImageState
+sealed trait Merged extends ImageState
+sealed trait Unmerged extends ImageState
```

Comment:
A bit unsure what having these `Merged` / `Unmerged` types give us, given that only `UnmergedImage` and `MergedImage` subtype them, so we can also use those types in place of these currently. Is there a plan for them in a future PR?

Reply:
Yeah, they seemed like they were going to be used but I'll remove them for now, can always add them back later

Reply:
Cool, suspected it might have been something like that :+1: